cypress/support/step_definitions/event.js

```diff
@@ -5,3 +5,24 @@ When('I click on {string}', (templateSelector) => {
   const selector = nunjucks.renderString(templateSelector, cy.context);
   cy.get(selector).click();
 });
+
+When('I double click on last element of {string}', (templateSelector) => {
```

Comment:
why last ? not working on get() ?

Reply:
I had trouble targetting a specific element.
Done - remove this rule